docs/cli/index.md

```diff
@@ -1293,6 +1294,49 @@ starport tools protoc -- --version
 
 * [starport tools](#starport-tools)	 - Tools for advanced users
 
+## starport tools completions
+
+Generates a completion script. 
+
+**Synopsis**
+
+The ``completions`` command generates a completion script to be used in your shell. However, the generated script depends on
+[bash-completion](https://github.com/scop/bash-completion) being installed and enabled in your system. Most of *nix like
+operating systems come with it by default and we assume it is already installed and operational. 
```

Comment:
```suggestion
The ``completions`` command generates a completion script you can use in your shell. Using the `completions` command to generate a completion script requires that [bash-completion](https://github.com/scop/bash-completion) is installed and enabled in your system. Since most Unix-like operating systems come with bash-completion by default, bash-completion is probably already installed and operational. 
```

Reply:
@andraderaf and @fadeev 
is there a way I can limit line length when I do a multi-line suggestion like this one? 
I know we are trying to limit line length? (or what is it that I can do differently here to make a PR commit of this suggestion clean?)

Reply:
I don't think we should be limiting line length, at least it MD files (CLI might be another story). Users' text editors are capable of wrapping lines.

Reply:
I think saying "Using the `completions` command to generate a completion script requires that [bash-completion](https://github.com/scop/bash-completion) is installed and enabled in your system..." is technically inaccurate, since the command starport tool completions will work whether or not bash_completions is installed. When running the command, it outputs a functional script. That resulting script is the one that depends on bash_completions to work.

Reply:
> I don't think we should be limiting line length, at least it MD files (CLI might be another story). Users' text editors are capable of wrapping lines.

oh thanks @fadeev I am glad my feedback won't interfere then. I recall a Slack comment you made (a few weeks ago, I think?) about using single lines in PRs. What single lines should I be aware of?